backend/devices/bitbox/keystore.go

```diff
@@ -132,3 +141,31 @@ func (keystore *keystore) SignTransaction(proposedTx coin.ProposedTransaction) e
 	}
 	return nil
 }
+
+func (keystore *keystore) SignETHTransaction(txProposal *eth.TxProposal) error {
```

Comment:
Make method private. (And personal preference: Avoid forward reference in `SignTransaction`, i.e. move this method above `SignTransaction` – but we're not consistent anyway and I can live without such a change.)

Reply:
Done.